### ISSUE TITLE
Sampling Instrument name for imaging instrument [BODCNVS-1365]

### ISSUE THREAD

**Issue by PatriciaCabrera** [nvs-vocabs/P01#275]
I don't know where to place this issue, since I don't find a Q01 repository, so please move it where this fits better. 

Last year we requested the creation in L22 of several imaging instruments. Because sometimes this instruments are benchtops, and require the collection of the sample first, we are using the term "Sampling Instrument name" (http://vocab.nerc.ac.uk/collection/Q01/current/Q0100002/) for the device to collect the sample, but we don't find a term to specify the imaging instrument. Would it be possible to create "Imaging Instrument name", or "Processing instrument name" if more general and convenient in this Q01 collection?  Thanks!

**Comment by LynnDelgat** [comment]
Yes, @rubenpp7 and I both agree that "Name of analytical instrument" would be an appropriate name as well.

**Comment by hanrev** [comment]
This new P01 term has now been loaded and will be live on the NVS tomorrow.
codval: NMANINST	
codnam: Name of analytical instrument
definition: Name of the equipment used to qualitatively and/or quantitatively measure the composition, properties, structure of substances or samples e.g. microscope, scale, caliper, fish measuring board and more.

**Comment by hanrev** [comment]
The term is now live on the NVS: http://vocab.nerc.ac.uk/collection/P01/current/NMANINST/
